utils/utils.php

```diff
@@ -61,10 +61,26 @@ function tlms_isApiKey($apiKey){
 	}
 }
 
+if(!function_exists('tlms_parseDate')){
+	function tlms_parseDate($format, $date){
+		$isPM = (stripos($date, 'PM') !== false);
+		$date = str_replace(array('AM', 'PM'), '', $date);
+
+		$date = DateTime::createFromFormat(trim($format), trim($date));
+
+		if ($isPM){
+			$date->modify('+12 hours');
+		}
```

Comment:
Fixed in https://github.com/epignosis/talentlms-wordpress/pull/3/commits/4f5e80b770e0d306d6fa0b87c7ead668328a2c41

Reply:
I just realised that something similar happens to AM: '16-12-2021 12:01 AM' will be converted it to ''16-12-2021 12:01' instead of '16-12-2021 00:01'. I suggest something like
```
$isAM = (stripos($date, 'AM') !== false);
if ($isAM && $date->format('H') === '12'){
    $date->modify('-12 hours');
}
```

Reply:
Fixed in https://github.com/epignosis/talentlms-wordpress/pull/3/commits/8c4d7cec9c1a2492c2d98431952be8a3fbcf3794

Reply:
I think that if the incoming hour is 12:00 in 24-hour format this will not ork properly since it will convert it to 00:00

Reply:
I now do the parsing only when we are using the 12hour format so we should be ok. Fixed in https://github.com/epignosis/talentlms-wordpress/pull/3/commits/9f335d99a408011abb5c1440749228aef6ca287a